v2/metadata/client.go

```diff
@@ -313,6 +315,16 @@ func (c *Client) CreateExecution(ctx context.Context, pipeline *Pipeline, taskNa
 	}, nil
 }
 
+// GetExecutions ...
+func (c *Client) GetExecutions(ctx context.Context, ids []int64) ([]*pb.Execution, error) {
```

Comment:
It's worth adding a wrapper for GetExecution too, because checking response length == 1 can be shared

Reply:
It depends. Sometimes it is expected that the response length is equal to 0. How the caller handles  response length is equal to 0 could be different.

Reply:
Ohh, that makes sense